index.js

```diff
@@ -0,0 +1,104 @@
+import { addMenuLinks } from './modules/displaymenu.js';
+import { displayTime } from './modules/date.js';
+import Book from './modules/class-book.js';
+
+const removeBookFromDOM = (listBooks, id) => {
+  if (id !== -1) {
+    listBooks.removeBook(id);
+  }
+};
+
+const addRemoveButtons = (listBooks) => {
+  const buttons = document.getElementsByClassName('btn remove-btn');
+  for (let i = 0; i < buttons.length; i += 1) {
+    buttons[i].addEventListener('click', () => {
+      removeBookFromDOM(listBooks, i);
+    });
+  }
+};
+
+class Library {
+  constructor() {
+    this.books = [];
+  }
+
+  saveToLocalStorage() {
+    localStorage.setItem('MY-Library', JSON.stringify(this.books));
+  }
+
+  getDataFromLocalStorage() {
+    try {
+      const data = JSON.parse(localStorage.getItem('MY-Library'));
+      if (data !== null) {
+        this.books = data;
+      }
+    } catch (error) {
+      this.saveToLocalStorage();
+    }
+  }
+
+  addBook() {
```

Comment:
- [ ] Kindly consider using ES6 arrow functions in this class for all the methods instances as you have used ES6 arrow methods outside this class as it is project requirement to use ES6 arrow functions.